packages/loot-core/src/platform/client/fetch/index.d.ts

```diff
@@ -41,7 +41,8 @@ export type Unlisten = typeof unlisten;
 /** Mock functions */
 export function initServer(handlers: {
   query: (query: { table: string; selectExpressions: unknown }) => Promise<{
-    data: unknown;
```

Comment:
Just curious, what issue was the unknown causing?

Reply:
This particular instance didn't cause any error. The other one in `server-handlers.d.ts` causes a bunch of errors like `Type 'unknown' is not assignable to type 'number'` because the result type of a query varies wildly based on the query being run. We could totally have better types here in the future!

Happy to revert the mock if you like, but my thinking is it would be good to keep the mock and implementation types the same